OpenRA.Mods.Common/Widgets/Logic/CreditsLogic.cs

```diff
@@ -39,7 +40,9 @@ public CreditsLogic(Widget widget, ModData modData, Action onExit)
 				onExit();
 			};
 
-			engineLines = ParseLines("AUTHORS");
+			engineLines = ParseLines(
+				File.ReadAllLines(Platform.ResolvePath(Path.Combine(".", "AUTHORS")))
```

Comment:
The `Path.Combine` here is not needed. `Platform.ResolvePath` is designed to work with unix-style path separators.

Reply:
Also,

> Widgets\Logic\CreditsLogic.cs(65,5): error SA1009: Closing parenthesis should not be preceded by a space. [C:\projects\openra\OpenRA.Mods.Common\OpenRA.Mods.Common.csproj]